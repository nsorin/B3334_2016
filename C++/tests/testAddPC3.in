LIST
PC pc1 0 0 -2 2 0 5 3 4 4 1
LIST
EXIT
