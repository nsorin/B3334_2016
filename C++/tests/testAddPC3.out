OK
PC pc1 0 0 -2 2 0 5 3 4 4 1
